src/org/daisy/dotify/formatter/impl/page/PageSequenceBuilder2.java

```diff
@@ -298,24 +300,47 @@ private PageImpl nextPageInner(
             boolean wasSplitInSequence,
             boolean isFirst
     ) throws PaginatorException, RestartPaginationException { // pagination must be restarted in
-        // PageStructBuilder.paginateInner
+                                                              // PageStructBuilder.paginateInner
         PageImpl current = newPage(pageNumberOffset);
-        if (
-            pcbl != null &&
-            // Store a mapping from the BlockLineLocation of the last line of the page before the
-            // previous page to the BlockLineLocation of the last line of the previous page. This
-            // info is used (in the next iteration) in SheetDataSource to obtain info about the
-            // verso page of a sheet when we are on a recto page of that sheet.
+        // whether this is the last page of the volume and a volume transition was provided
+        boolean interruptContentPresent =
+            context.getTransitionBuilder().getProperties().getApplicationRange() != ApplicationRange.NONE &&
             transitionContent.isPresent() &&
-            transitionContent.get().getType() == TransitionContent.Type.INTERRUPT
-        ) {
-
-            blockContext.getRefs().setNextPageDetailsInSequence(pcbl, current.getDetails());
-        }
+            transitionContent.get().getType() == TransitionContent.Type.INTERRUPT;
         if (nextEmpty) {
+            // Return an empty page because the next row should be on a new sheet and we are
+            // currently on the back of a sheet.
```

Comment:
Is this also valid in simplex mode?

Reply:
Yes. In simplex mode nextEmpty will never be true. I'll change the comment a bit.

Reply:
See https://github.com/bertfrees/dotify.library/commit/0c95dc06ae7c21fa536560254e3d10df86818cf7